dev-scheme/racket/racket-8.2-r1.ebuild

```diff
@@ -135,30 +137,27 @@ src_install() {
 }
 
 pkg_preinst() {
-	# If we are merging the same version (NOT revision!)
-	# check if pkg database files exist and do not overwrite them
-	local pvr rel
-	for pvr in ${REPLACING_VERSIONS}; do
-		rel="${pvr%-r*}"
-		if [[ "${rel}" == "${PV}" ]]; then
-			echo "We are installing the same version: ${rel}"
-			local rktd
-			for rktd in "${PKGDB[@]}"; do
-				if [[ -f "${EROOT}/${rktd}" ]]; then
-					einfo "Keeping old file: ${rktd}"
-					mv "${ED}"/${rktd} "${ED}"/${rktd}.bak ||
-						die "failed to create a backup of ${rktd}"
-					cp "${EROOT}"/${rktd} "${ED}"/${rktd} ||
-						die "failed to create a copy of ${rktd}"
-				fi
-			done
-		fi
-	done
+	# If we are merging the same SLOT check if package
+	# database files exist and do not overwrite them
+	if has_version "${CATEGORY}/${PN}:${SLOT}"; then
```

Comment:
I kind of wonder if this should be CONFIG_PROTECTed but I suppose users won't be touching it.

Reply:
they are not very friendly to edit, so I dont see a reason for CONFIG_PROTECT here

Reply:
ack

Reply:
yea, thats the strange think it looks for readline on my system

Reply:
https://github.com/racket/racket/blob/master/pkgs/racket-doc/scribblings/more/more.scrbl#L106

So it can used both, but I do not know by it does not look fro editline on my system (even when I unmerged readline)

Reply:
Weird.